src/Pimple/Psr11/Container.php

```diff
@@ -38,16 +38,25 @@ final class Container implements ContainerInterface
 {
     private $pimple;
 
+    /**
+     * @param PimpleContainer $pimple The Container instance used to locate services
+     */
```

Comment:
This can be removed as it does not add any value that is not already part of the PHP code